src/test/kotlin/BlackjackGameTest.kt

```diff
@@ -64,6 +62,24 @@ internal class BlackjackGameTest {
                     )
                 ).isTrue()
             },
+            {
+                assertThat(
+                    CardVendor().isGetExtraCard(
+                        listOf(
+                            Card(
+                                CardNumber.THREE,
+                                CardShape.HEARTS
+                            )
```

Comment:
```suggestion
                            Card(
                                CardNumber.TEN,
                                CardShape.HEARTS
                            ),
                            Card(
                                CardNumber.TEN,
                                CardShape.HEARTS
                            )
```
가급적이면 경계선에 해당하는 값으로 테스트 코드를 작성하는 것이 좋습니다. (이하 관련 내용 동일)
이 테스트 코드에는 20이 되겠네요.